model/special_clause.py

```
@@ -1,5 +1,848 @@
+        for key, value in tenant_survey.items():
+            context += f"- {key}: {value}\n"
+        
+        return context
+
+    def _prepare_existing_context(self, existing_clauses: List[str]) -> tuple:
+        """기존 계약서 컨텍스트 준비"""
+        existing_context = ""
+        info_clause = None
+        
+        if existing_clauses:
+            existing_context = f"\n# 기존 계약서의 특약들:\n"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)